backend/src/test/java/ch/puzzle/okr/service/business/ActionBusinessServiceTest.java

```diff
@@ -170,4 +171,64 @@ void shouldDeleteAction() {
         actionBusinessService.deleteEntityById(action1.getId());
         verify(actionPersistenceService, times(1)).deleteById(action1.getId());
     }
+
+    @ParameterizedTest(name = "Should return a empty list when actions are null or empty")
```

Comment:
```suggestion
    @ParameterizedTest(name = "Should return a empty list when actions are {0}")
```
Makes this a tiny bit nicer 
![image](https://github.com/user-attachments/assets/c44daeeb-4255-4f80-ad09-ad5657e905de)